backend/core/src/auth/auth.module.ts

```diff
@@ -38,7 +38,7 @@ import { UserProfileController } from "./controllers/user-profile.controller"
     EmailModule,
   ],
   providers: [LocalStrategy, JwtStrategy, AuthService, AuthzService, UserService, PasswordService],
-  exports: [AuthzService, AuthService, UserService],
+  exports: [AuthzService, AuthService, UserService, PasswordService],
```

Comment:
I couldn't get it to build without this:

```
BACKEND_CORE [Nest] 1573917   - 11/03/2021, 3:20:19 PM   [ExceptionHandler] Nest can't resolve dependencies of the UserService (UserRepository, ApplicationRepository, EmailService, ConfigService, AuthService, AuthzService, ?, JurisdictionResolverService). Please make sure that the argument PasswordService at index [6] is available in the SmsModule context.
```
Since `PasswordService` is a dependency of `UserService`, and `SmsService` depends on `UserService`, it needs to be exported from somewhere, I think.

Reply:
This actually isn't needed anymore, since I removed the UserService dependency from SmsService.